src/components/AvatarGroup/AvatarGroup.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { createUniDriverFactory } from 'wix-ui-test-utils/uni-driver-factory';
+import { isUniEnzymeTestkitExists } from 'wix-ui-test-utils/enzyme';
+import { isUniTestkitExists } from 'wix-ui-test-utils/vanilla';
+import { mount } from 'enzyme';
+import { avatarGroupDriverFactory } from './AvatarGroup.driver';
+import { AvatarGroup } from './';
+import { avatarGroupTestkitFactory } from '../../testkit';
+import { avatarGroupTestkitFactory as enzymeAvatarGroupTestkitFactory } from '../../testkit/enzyme';
+
+describe('AvatarGroup', () => {
+  const createDriver = createUniDriverFactory(avatarGroupDriverFactory);
+
+  it('should render', async () => {
+    const driver = createDriver(<AvatarGroup items={[]} />);
+    expect(await driver.exists()).toBe(true);
+  });
+
```

Comment:
I just thought that it would be nice to have a test to see that the Avatar never shows more than the `max` number of avatars.
It's good as a guard, so that if anyone makes changes in the future, we can be assured that this behaviour will not break.

Reply:
I'm kind of checking this in visual tests. But maybe it makes sense to add it here as well :)

Reply:
Agree, adding tests :)

Reply:
Added;